community-containers/jellyseerr/jellyseerr.json

```
@@ -12,12 +12,14 @@
                 {
                     "ip_binding": "%APACHE_IP_BINDING%",
                     "port_number": "5055",
-                    "protocol": "tcp"
+                    "protocol": "tcp",
+                    "host_port": "5055"
                 }
             ],
             "environment": [
                 "PORT=5055",
-                "TZ=%TIMEZONE%"
+                "TZ=%TIMEZONE%",
+                "TINI_SUBREAPER="
```

Instead of doing this, you could simply set init to false like so: https://github.com/nextcloud/all-in-one/blob/3a2b1a2efefd7e083fefe081d888df1a72663688/community-containers/pi-hole/pi-hole.json#L11

```suggestion
                "TZ=%TIMEZONE%"
```